javascript/javascript1/week2/studentManager.js

```diff
@@ -0,0 +1,35 @@
+const class07Students = [];
+function addStudentToClass(studentName) {
+  if (getNumberOfStudents(class07Students) < 6) {
+    if (studentName == " ") {
+      output = "You cannot add an empty string to a class";
+      return output;
+    } else {
+      for (let i = 0; i < getNumberOfStudents(class07Students); i++) {
+        if (studentName == class07Students[i]) {
+          output = "student " +  studentName  + "is already in the class";
+          return output;
+        }
+      }
```

Comment:
you can check if the student already exists easier using `includes()` which returns a boolean value. Example:
```js
[1, 2, 3].includes(1); // true
[1, 2, 3].includes(4); // false
```